src/main/kotlin/uk/gov/justice/digital/assessments/services/EpisodeService.kt

```diff
@@ -116,6 +117,29 @@ class EpisodeService(
     return newEpisode
   }
 
+  fun removeOrphanedAnswers(episode: AssessmentEpisodeEntity) {
```

Comment:
No more so than questions and answers. Happy to keep this out of the database if it's something we want to avoid. Regarding the complexity my intent was to extend this to the `yes/no` questions with conditional `details` questions and eventually to the modern slavery piece

Reply:
Sorted - Simplified this function